### PR TITLE
One time fee change

### PR DESCRIPTION
**Pull request by raoaliahmad**

### CODE REVIEW

File: web/src/actions/index.js [hunk 1]
```diff
@@ -251,6 +251,25 @@ export const addMultiplePayments = (payments) => dispatch => {
...
+			amount : f.amount,
+			name : f.name,
+			period: f.period,
+			type: f.type
+		}
+	}))
+
+	console.log("MERGES", merges);
+
+	///dispatch(createMerges(merges))
```
Comment (taimur38) [inline web/src/actions/index.js]: dispatch commented out

File: web/src/modules/Student/ManageFees.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+				active: true,
+				good: true,
+				text: "Saved!"
+			}
+		})
+
+		setTimeout(() => this.setState({ banner: { active: false } }), 3000);
+	}
+
+	validateFilter = () => this.state.feeFilter === "to_all_students" ? this.setState({ section_id: "" }) : true
```
Comment (taimur38) [inline web/src/modules/Student/ManageFees.js]: this isn't a filter validation function, it's a callback that is triggered when filter is changed

File: web/src/modules/Student/ManageFees.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+						<label>Add To</label>		
+							<select {...this.former.super_handle(["feeFilter"], () => true, this.validateFilter)}>
+									<option value="">Select Students</option>
+									<option value="to_all_students">All Students</option>
+									<option value="to_single_class">Single Class</option>
+							</select>
+						</div>
+						
+						{this.state.feeFilter === "to_single_class" ?  //Section Wise
+                        <div className="row">
```
Comment (taimur38) [inline web/src/modules/Student/ManageFees.js]: tab spacing

File: web/src/modules/Student/ManageFees.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+								<option value="" >Select</option>
+								{Object.entries(getSectionsFromClasses(classes))
+									.map(([id, C]) => <option key={id} value={C.id}>{C.namespaced_name}</option>)
+								}
+							</select>
+						</div> : false}
+					</div>
+
+					<div className="section">
+                        <div className="row">
```
Comment (taimur38) [inline web/src/modules/Student/ManageFees.js]: also spacing

File: web/src/modules/Student/ManageFees.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+                            <label>Name</label>		
+                            <input type="text" {...this.former.super_handle(["fee","name"])} placeholder="Enter Name"/>
+						</div>
+                        <div className="row"> 
+                            <label>Amount</label>		
+                            <input type="text" {...this.former.super_handle(["fee","amount"])} placeholder="Enter Amount"/>
+						</div>
+                        <div className="row">
+                        <label>Fee Period</label>		
+							<select {...this.former.super_handle(["fee","period"])} value={this.state.smsFilter}>
```
Comment (taimur38) [inline web/src/modules/Student/ManageFees.js]: smsFilter??

File: web/src/modules/Student/ManageFees.js (hunk 2 above)
Comment (raoaliahmad) [reply web/src/modules/Student/ManageFees.js]: I've changed the name to filterCallBack

File: web/src/modules/Student/ManageFees.js (hunk 5 above)
Comment (raoaliahmad) [reply web/src/modules/Student/ManageFees.js]: Dumb  mistake :D

**Pull request merged**